dump_utils.cpp

```diff
@@ -82,9 +82,9 @@ void createPEL(
     }
 }
 void createPELOnDumpDelete(sdbusplus::bus::bus& dBus,
-                           const std::string& dumpFilePath,
+                           const std::string dumpFilePath,
                            const std::string& dumpFileType,
-                           const std::string& dumpId, const std::string& pelSev,
+                           const std::string dumpId, const std::string& pelSev,
                            const std::string& errIntf)
```

Comment:
should be changed to pass by value as this is called from async method to be consistent also add some comment stating this is called from async method or else someone in the future will change it to &